data/pages-content/sponsor/information.md

```diff
@@ -134,10 +138,16 @@ Shipping and customs for your goods are handled by <a href = "https://www.interf
 
 Here are shipping information provided by Interflow:
 
-- Email <a href = "mailto:niall@interflow.ie">Interflow</a> for Shipping Guidelines
+-  <a href = "https://drive.google.com/file/d/1zFCopPlWoC9ToPEmlxFessCJ4UaxLqu_/view?usp=sharing" target="_blank">Shipping Guidelines</a>. <b>Important: Please provide Interflow your Company name, point of contact name and email.</b> Contact Interflow for Details.
+- <a href = "https://www.interflow.ie/freight-order-form/" target="_blank">Freight Order link</a>.
 - <a href = "https://drive.google.com/file/d/1TtlW5nWrOfuK-5JgK6iKq1S1rDAF2U6_/view?usp=sharing" target="_blank">Delivery address and Shipping Label</a>.
 
-Earliest delivery date: TBD. Please contact Niall or Anderson directly for more details.
+**Deadlines:**
+- Deadline for receiving to the warehouse: 2022-07-06
+- Deadline for orders: 2022-06-30
+
+If you have any last-minute orders, please contact Interflow and keep them in the loop.
+Company name, point of contact name and email would be perfect
```

Comment:
I'd remove this line "Company name, point of contact name and email would be perfect" possibly copy/paste mistake?

Reply:
hahaha thank you! Fixed!